samples/customer-registry-quickstart/src/main/resources/application.conf

```diff
@@ -0,0 +1,7 @@
+akkaserverless {
+  system {
+    loggers = ["akka.event.slf4j.Slf4jLogger"]
+    loglevel = "DEBUG"
+    logging-filter = "akka.event.slf4j.Slf4jLoggingFilter"
+  }
+}
```

Comment:
Created issue https://github.com/lightbend/akkaserverless-java-sdk/issues/210 for this